scripts/menu_helper_functions/menu_helper_functions.gml

```diff
@@ -0,0 +1,9 @@
+function createMenu(arr){
+	with (instance_create_layer(x,y,layer,obj_menu))
```

Comment:
In 2022.5 you can add pre-create values as a struct. [Check out the docs here.](https://manual.yoyogames.com/GameMaker_Language/GML_Reference/Asset_Management/Instances/instance_create_layer.htm)

Example:
```
var inst = instance_create_layer(x, y, "Instances", obj_bullet,
{
    speed : shoot_speed,
    direction : image_angle
});
```

Reply:
**!!This is not a change request!!**